client/me/index.js

```diff
@@ -15,5 +22,12 @@ export default function () {
 	page( '/me/trophies', controller.profileRedirect, makeLayout, clientRender );
 	page( '/me/find-friends', controller.profileRedirect, makeLayout, clientRender );
 
-	page( '/me/get-apps', controller.sidebar, controller.apps, makeLayout, clientRender );
+	page(
+		'/me/get-apps',
+		controller.sidebar,
+		setSelectedSiteIdByOrigin,
```

Comment:
Why do we need this here 🤔

Reply:
I don't think we need this 🤦 

@fushar WDYT about removing middleware from `/me` also since we don't have a link to /me on wp-admin (there's a link to /profile.php instead).

Reply:
We need that on `/me` because that's where the `Edit Profile` menu links to, on Default-view sites 😄

Reply:
Yes, but on Default interface, it stores the selected site in memory, so it should not need `origin_site_id`.

Reply:
Ohhhh you're right!!! No need then 😄

Reply:
Let's remove it and see how it goes!